src/theme/components/fieldset.ts

```diff
@@ -18,6 +18,9 @@ const Fieldset = defineStyleConfig({
         color: "ui.typography.heading",
         ...labelLegendText,
         ...screenreaderStyles,
+        span: {
+          fontWeight: "light",
```

Comment:
The PR description says "medium" but this is "light". Is this correct?

Reply:
Sorry. That was a typo in the PR description. It should say `light` to match the description in the Jira ticket.

Reply:
I updated the PR description.